### DISCUSSION TITLE
Windows does not show up when installing

### DISCUSSION THREAD

**Discussion by SomewhatAwake** [BlackArch/blackarch#3933] [Support] [answered]
Type: Question

## Question
When installing BlackArch via the netinstall iso on a USB, once I'm at the partition drives section of the installer and select my drive, it shows no partitions on the drive, only free space, despite Windows already being on the drive.

**Comment by anunna** [comment, marked as answer]
You can create the necessary partitions for BlackArch (root/swap) manually with `fdisk` and then skip the `cfdisk` step and set your mount points. And you obviously set the boot mount on the existing efi partition created by Windows. 

I'd recommend first testing this on a mock system.

**Reply by SomewhatAwake** [reply]
Thanks for the tip. Will do
